src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -1,53 +1,23 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{3F1F2F50-AF1A-4A5A-BEDB-193372F068D7}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Xamarin.Android.Tasks</RootNamespace>
-    <AssemblyName>Xamarin.Android.Build.Tasks</AssemblyName>
-    <FileAlignment>512</FileAlignment>
-    <SchemaVersion>2.0</SchemaVersion>
-    <LibZipSharpBundleAllNativeLibraries>true</LibZipSharpBundleAllNativeLibraries>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <DebugSymbols>True</DebugSymbols>
-  </PropertyGroup>
+<Project>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Comment:
Did you need to workaround something by importing in a specific order? Or could this have `<Project Sdk="Microsoft.NET.Sdk">`?

Reply:
This was done to work around the issue described in https://github.com/microsoft/msbuild/pull/4922